src/current/v24.1/cutover-replication.md

```diff
@@ -154,16 +154,115 @@ At this point, the primary and standby clusters are entirely independent. You wi
+
+    {{site.data.alerts.callout_success}}
+    For details on connection strings, refer to the [Connection reference]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}#connection-reference).
+    {{site.data.alerts.end}}
+
+1. Check the status of the virtual cluster on **A**:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SHOW VIRTUAL CLUSTER {vc_a};
+    ~~~
+
+    {% include_cached copy-clipboard.html %}
+    ~~~
+     id |  name  |     data_state     | service_mode
+    ----+--------+--------------------+---------------
+      1 | system | ready              | shared
+      3 | {vc_a} | replicating        | none
+      4 | test   | replicating        | none
+      (2 rows)
+    ~~~
+
+1. Run the following from **cluster A** to start the cutover:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    ALTER VIRTUAL CLUSTER {vc_a} COMPLETE REPLICATION TO LATEST;
+    ~~~
+
+    The `cutover_time` is the timestamp at which the replicated data is consistent. The cluster will revert any data above this timestamp:
+
+    ~~~
+               cutover_time
+    ----------------------------------
+      1714497890000000000.0000000000
+    (1 row)
+    ~~~
+
+1. From **cluster A**, bring the virtual cluster online with:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    ALTER VIRTUAL CLUSTER {vc_a} START SERVICE SHARED;
+    ~~~
+
+1. To make **cluster A's** virtual cluster the default for connection strings, set the following [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}):
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SET CLUSTER SETTING server.controller.default_target_cluster='{vc_a}';
+    ~~~
 
-1. Cluster B is now serving application traffic after the cutover.
-1. Drop the application virtual cluster from the cluster A with `DROP VIRTUAL CLUSTER`. {% comment %}link here{% endcomment %}
-1. Start a replication stream that sends updates from cluster B to cluster A. Refer to [Start replication]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}#step-4-start-replication).
+At this point, **cluster A** is once again the primary and **cluster B** is once again the standby. The clusters are entirely independent. You will need to use your own network load balancers, DNS servers, or other network configuration to direct application traffic to the primary (**cluster A**). To enable physical cluster replication again, from the primary to the standby (or a completely different cluster), refer to [Set Up Physical Cluster Replication]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}).
```

Comment:
```suggestion
At this point, **cluster A** is once again the primary and **cluster B** is once again the standby. The clusters are entirely independent. To direct application traffic to the primary (**Cluster A**), you will need to use your own network load balancers, DNS servers, or other network configuration to direct application traffic to the primary (**cluster A**). To enable physical cluster replication again, from the primary to the standby (or a completely different cluster), refer to [Set Up Physical Cluster Replication]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}).
```
If Cluster B is once again the standby after the above, then I think the last sentence isn't needed. WDYT?

Reply:
Why? It is just pointing users back to the page to set up physical cluster replication, unless I am missing something.